eventlog/el/main.go

```diff
@@ -33,6 +44,16 @@ type bcConfig struct {
 	ByzCoinID skipchain.SkipBlockID
 }
 
+// Presets for -clientid and -clientsecret.
+var clientIDs = map[string]string{
```

Comment:
these should be in config files

Reply:
You can override them from the commandline. They are just helpers so that if you don't put them on the commandline, the right thing will still happen.